app/controllers/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+# class UsersController < ApplicationController
+#   def index; end
+
+#   def destroy
+#     @user.destroy
+#     respond_to do |format|
+#       format.html { redirect_to root_url, notice: 'User was successfully destroyed.' }
+#     end
+#   end
+# end
```

Comment:
- [ ] Please remove any unused code for a cleaner and more optimized code base. And if you're not using the whole file of code, I recommend deleting it. 🙏